readmes/01_preface.md

```diff
@@ -42,57 +42,58 @@ When we say Linux, we are referring to a type of operating system, and usually t
- 
-## Workflow 
- 
+
+## Workflow
+
 You'd think caveat #2 would be a **huge** problem, right?
 
 It's actually not. It's all in the _workflow_!
- 
-Essentially, we use the Subsystem for two things: 
- 
-1. To install and run software that only ran in POSIX environments. 
-1. To give us a Linux shell that allows us to run POSIX utilities, services, and commands. 
 
-The vast majority of issues with Web Development on Windows stem from not being able to run software in POSIX environments, and the inability to use many of the utilities and commands that come with a POSIX-Compliant Shell. Since the Linux Subsystem is directly mounted to your Windows File System (W-FS), it has access to all of your Windows files. That means the Subsystem can use those files, and can successfully run them through software in it's POSIX environment. 
+Essentially, we use the Subsystem for two things:
+
+1. To install and run software that only ran in POSIX environments.
+2. To give us a Linux shell that allows us to run POSIX utilities, services, and commands.
+
+The vast majority of issues with Web Development on Windows stem from not being able to run software in POSIX environments, and the inability to use many of the utilities and commands that come with a POSIX-Compliant Shell. Since the Linux Subsystem is directly mounted to your Windows File System (W-FS), it has access to all of your Windows files. That means the Subsystem can use those files, and can successfully run them through software in its POSIX environment.
+
+This gives you the best of both worlds in regard to the Windows Shell and Linux Shell. There are still some features that aren't available, but with this now being around for the last 20 months, much of what you need is there, and more are being developed.
+
+The typical workflow looks like this:
 
-This gives you the best of both worlds in regards to the Windows Shell and Linux Shell. There are still some features that aren't available, but with this now being around for the last 20 months, much of what you need is there, and more are being developed.
- 
-The typical workflow looks like this: 
 1. Open whichever Linux distro you are using, which is just like opening a terminal window.
-1. Work normally, doing everything on the Windows FS.  
-1. When you need new software that is only available for Linux, navigate to the Linux Subsystem's FS and install that software through the command line in the very same terminal window. 
-1. Then, while also still in the same terminal window, navigate back to the Windows FS and use the software as desired. 
-- Since the software is connected through the PATH, it runs just fine. 
-- Since the Linux Subsystem also has access to your Windows files, it can use them with it's software just fine as well. 
+2. Work normally, doing everything on the Windows FS.
```

Comment:
The 2 3 and 4 here don't need to be changed. You can leave them as 1 and markdown will automatically fill them in with the correct numbers.